library/java/net/openid/appauth/TokenRequest.java

```diff
@@ -185,6 +191,14 @@
     public final String scope;
 
     /**
+     * String value used to associate a Client session with an ID Token, and to mitigate replay attacks.
```

Comment:
Hello William,
Thanks for the review.
I will remove nonce from the Token Request.